lib/hydra/pcdm.rb

```diff
@@ -14,6 +14,7 @@ module PCDM
     autoload :Collection,             'hydra/pcdm/models/collection'
     autoload :Object,                 'hydra/pcdm/models/object'
     autoload :File,                   'hydra/pcdm/models/file'
+    autoload :VersionedFile,          'hydra/pcdm/models/versioned_file'
```

Comment:
My biggest question about #115 is whether the VersionedFile model should live in hydra-pcdm or hydra-works, with a sneaking suspicion that it should be in hydra-works (since "versioned files" aren't a PCDM concept).  I'm sure you put this in hydra-pcdm for a reason though so I'd be happy to hear it!